.spellcheckwordlist.txt

```diff
@@ -537,7 +536,6 @@ provider
 ps
 psql
 public
-python
```

Comment:
I think people depend in this probably? Of course they could add it in webimage_extra_packages, but it seems like a normal thing to have.